services/movie.py

```diff
@@ -0,0 +1,49 @@
+from db.models import Movie
+from django.db.models import QuerySet
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None
+               ) -> QuerySet:
+    movies = Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        movies = Movie.objects.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        )
+
+    if not genres_ids and actors_ids:
+        movies = Movie.objects.filter(
+            actors__id__in=actors_ids
+        )
+
+    if genres_ids and not actors_ids:
+        movies = Movie.objects.filter(
+            genres__id__in=genres_ids,
+        )
```

Comment:
You can simplify this code
```suggestion
    movies = Movie.objects.all()
    if genres_ids:
        movies = movies.filter(
            genres__id__in=genres_ids,
        )

    if actors_ids:
        movies = movies.filter(
            actors__id__in=actors_ids
        )
```